### DISCUSSION TITLE
How to see which unit tests is running which line of production code

### DISCUSSION THREAD

**Discussion by buzzbonner** [FortuneN/FineCodeCoverage#351] [General]
Hi,

I'm using your tool and it's great!! However, I just have this one issue where I'd like to see what unit test relates to the green tested code bar indication in the VS code editor?

**Comment by tonyhallett** [comment]
The coverage tools that instrument your code that we invoke do not supply such information. 

You can visual studio to some degree for what you want.

**Comment by buzzbonner** [comment]
Thank you for the quick response
